src/ui/screens/AccountsScreen.tsx

```diff
@@ -201,31 +192,29 @@ const Buttons = ({navigation}: {navigation: Props['navigation']}) => {
   const {open} = state;
 
   return (
-    <Portal>
-      <FAB.Group
-        visible={true}
-        open={open}
-        icon={open ? 'minus' : 'plus'}
-        actions={[
-          {
-            icon: 'import',
-            label: 'Import seed',
-            onPress: () => navigation.navigate(importAccountScreen),
-          },
-          {
-            icon: 'plus',
-            label: 'Add External Account',
-            onPress: () => navigation.navigate(addAccountScreen),
-          },
-          {
-            icon: 'key-plus',
-            label: 'Generate New Seed',
-            onPress: () => navigation.navigate(mnemonicScreen),
-            small: false,
-          },
-        ]}
-        onStateChange={onStateChange}
-      />
-    </Portal>
+    <FAB.Group
+      visible={true}
+      open={open}
+      icon={open ? 'minus' : 'plus'}
+      actions={[
+        {
+          icon: 'import',
+          label: 'Import seed',
+          onPress: () => navigation.navigate(importAccountScreen),
+        },
+        {
+          icon: 'plus',
+          label: 'Add External Account',
+          onPress: () => navigation.navigate(addAccountScreen),
+        },
+        {
+          icon: 'key-plus',
+          label: 'Generate New Seed',
```

Comment:
I think we should change the wording here, most users don't know what a seed is. Probably "Create New Account" would be better.

Reply:
May be also change `Import seed` to `Import account`?

Reply:
Yeah that would be better as well 👍🏼